atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueueUtils.java

```diff
@@ -29,7 +29,8 @@
 public final class SweepQueueUtils {
     public static final long TS_COARSE_GRANULARITY = 10_000_000L;
     public static final long TS_FINE_GRANULARITY = 50_000L;
-    public static final long CAS_TS = 1L;
+    public static final long READ_TS = 1L;
```

Comment:
@ashrayjain for thoughts on timestamps here - we should make sure we're not interfering with compactions on the queue tables with our choice of timestamps

Reply:
CAS is always done at timestamp 0 anyway, so just kept it 0 for everything else and then read at 1. Note that these are atlas timestamps, we don't specify cassandra timestamps anywhere directly.

Reply:
As long as we are not overriding c* timestamps, compactions should progress as usual.